test/helpers/components/codemodules/codemodules.go

```diff
@@ -35,13 +38,22 @@ func CheckRuxitAgentProcFileHasNoConnInfo(testDynakube dynakube.DynaKube) featur
 			Name:      csi.DaemonSetName,
 			Namespace: testDynakube.Namespace,
 		}).ForEachPod(func(podItem corev1.Pod) {
-			// /data/codemodules/1.273.0.20230719-145632/agent/conf/ruxitagentproc.conf
```

Comment:
please return back this line

Reply:
added comment with newer version https://github.com/Dynatrace/dynatrace-operator/pull/4953/commits/37cd70910c30f1f933eb53aa81c4840d5f9ab034